packages/salesforcedx-vscode-apex/snippets/apex.json

```diff
@@ -181,5 +181,135 @@
             "throw new ${1:Exception}($0)"
         ],
         "description": "Throw a new exception"
+    },
+    "List": {
+        "prefix": "List",
+        "body": [
+            "List<${1:object}> ${2:listName} = new List<$1>();"
+        ],
+        "description": "List constructor"
+    },
+    "map": {
+        "prefix": "Map",
+        "body": [
+            "Map<${1:object}, ${2:object}> ${3:mapName} = new Map<$1, $2>();"
+        ],
+        "description": "Map constructor"
+    },
+    "set": {
+        "prefix": "Set",
+        "body": [
+            "Set<${1:object}> ${2:setName} = new Set<$1>();"
+        ],
+        "description": "Set constructor"
+    },
+    "@AuraEnabled": {
+        "prefix": "@AuraEnabled",
+        "body": [
+            "@AuraEnabled",
+            "${1|public,global|} static ${2:string} ${3:methodName}(){",
+            "  $4",
+            "}"
+        ],
+        "description": "Aura Enabled Apex Method"
+    },
+    "userCreation": {
+        "prefix": "createUsers",
+        "body": [
+            "public static List<user> createUsers(integer numberOfUsers, string profileName){",
+            "  List<user> users = new List<user>();",
+            "  Profile p = [SELECT Id FROM Profile WHERE Name=:profileName];",
+            "  for (integer i=0; i<numberOfUsers; i++){",
+            "    string x = string.valueOf(i);",
+            "    User u = new User(Alias = 't'+x, Email='standarduser'+x+'@${2:someDomain.test}',",
+            "      EmailEncodingKey='UTF-8', LastName='Testing' + x, LanguageLocaleKey='en_US',LocaleSidKey='en_US', ProfileId = p.Id,",
+            "      TimeZoneSidKey='America/Los_Angeles', UserName='standarduser'+x+'@${2:someDomain.test}');",
+            "  users.add(u);",
+            "  }",
+            "",
+            "  insert users;",
+            "  return users;",
+            "}"
+        ],
+        "description": "Creates some amount of test users for a test method or for @TestSetup"
+    },
+    "AuraHandledException": {
+        "prefix": "auraHandledException",
+        "body": [
+            "throw new AuraHandledException('${1:exceptionText}');"
+        ],
+        "description": "Throw an exception for a lightning component to display"
+    },
+    "TestMethod": {
+        "prefix": "testMethod",
+        "body": [
+            "static testMethod void ${1:methodName}(){",
+            "  ",
+            "  test.startTest();",
+            "  system.RunAs(${2:contextuser}){",
+            "    ",
+            "  test.stopTest();",
+            "  ",
+            "}"
+        ],
+        "description": "Test method with runAs and stop/start"
+    },
+    "TestSetup": {
+        "prefix": "@TestSetup",
+        "body": [
+            "@testSetup static void ${1:makeData}(){",
+            "  ",
+            "}"
+        ],
+        "description": "Test setup for creating data before test methods run"
+    },
+    "createAdmin": {
```

Comment:
Let's remove this snippet for now. I am iffy on it and I would prefer to not have too many snippets first so that we can keep this lean. For your own purposes, you can set it up using https://code.visualstudio.com/docs/editor/userdefinedsnippets.

Reply:
Done.  [my userdefinedsnippets is where all these came from :) ]